single_level.py

```diff
@@ -90,22 +90,64 @@ def makeRow(resource):
 		else:
 			row.append("false")
 	
-	print row
+	print row	
 
 def main():
-	print "Creating a csv"
+	#Get list of resources
+	resourceIds = requests.get(repositoryBaseURL + "/resources?all_ids=true", headers=headers)
+	
+	#Creating csv
 	writer = csv.writer(open(spreadsheet, "wb"))
-	column_headings = ["title", "resource", "extent", "date", "language", "repository", "creator", "scope", "restrictions"]
+	column_headings = ["title", "publish", "resource", "extent", "date", "language", "repository", "creator", "scope", "restrictions"]
 	writer.writerow(column_headings)
-
-	print "Getting a list of resources"
-	resourceIds = requests.get(repositoryBaseURL + "/resources?all_ids=true", headers=headers)
-	print "Writing rows"
-	for resourceId in resourceIds.json():
-			resource = (requests.get(repositoryBaseURL + "/resources/" + str(resourceId), headers=headers)).json()
-			if resource["publish"]:
+	
+	#User input to refine functionality of script 
+	print "Welcome to DACSspace!"
+	response_1 = raw_input("Do you want DACSspace to include only published resources? (y/n): ")
+	response_2 = raw_input("Do you want to further limit the script by specific resource ids? (y/n): ")
```

Comment:
I think it might be more efficient to just ask the user to input IDs, or leave it blank if you want to export all. That way, you could simply evaluate whether or not the value exists below instead of string matching and then asking another question.